x/ccv/consumer/keeper/distribution.go

```diff
@@ -169,7 +200,8 @@ func (k Keeper) GetEstimatedNextFeeDistribution(ctx sdk.Context) (types.NextFeeD
 	fracParam := k.GetConsumerRedistributionFrac(ctx)
 	frac, err := sdk.NewDecFromStr(fracParam)
 	if err != nil {
-		return types.NextFeeDistributionEstimate{}, err
+		// ConsumerRedistributionFrac was already validated when set as a param
+		panic(fmt.Errorf("ConsumerRedistributionFrac is invalid: %w", err))
```

Comment:
This is used in query functions and probably should not cause a panic at this level.

It would be better to refactor this so that the caller can decide about panics.

Reply:
This is one of those panics where it will never happen unless our param validation is incorrect or the sdk param system is incorrect. It would be safe to leave the panic imo since it should never happen, or simply discard the error